db/schema.rb

```diff
@@ -10,52 +10,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_08_17_055131) do
+ActiveRecord::Schema[7.0].define(version: 2023_08_17_101227) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "comments", force: :cascade do |t|
-    t.bigint "user_id_id", null: false
-    t.bigint "post_id_id", null: false
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
     t.string "text"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["post_id_id"], name: "index_comments_on_post_id_id"
-    t.index ["user_id_id"], name: "index_comments_on_user_id_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
   create_table "likes", force: :cascade do |t|
-    t.bigint "user_id_id", null: false
-    t.bigint "post_id_id", null: false
+    t.bigint "user_id", null: false
```

Comment:
- [ ] **[OPTIONAL]** This is beyond the scope of this project, as it was handled in a previous project, but I would strongly suggest you rename this table column to `author_id`, instead of `user_id` to match the associations.

Reply:
but it is not specified in the ER diagram that's y I used the user_id instead
take a look at the Diagram please [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek_v1_1.md)

Reply:
I believe this is t[he guideline for the create data model project](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md). The ERD link there contains `author_id`.